FrontEnd/styles.css

```diff
@@ -47,4 +66,22 @@ body{
     margin: 27px 0 38px;
     text-align: center;
 }
-/*Copyright end*/
\ No newline at end of file
+/*Copyright end*/
+/*List start*/
+.list .item{
```

Comment:
It will be better if we can list out the full list of all class for .list component, some can be empty